packages/logger/test/e2e/logger/workerLogger.js

```diff
@@ -1,9 +1,10 @@
 import fs from "node:fs";
 import worker from "node:worker_threads";
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why is it extraneous? should have `@chainsafe/threads` as dev dependency of logger if it is required for testing